figures/figure-comparison-ERP-TRF.py

```diff
@@ -0,0 +1,180 @@
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.11.4
+#   kernelspec:
+#     display_name: tutorialAlice
+#     language: python
+#     name: tutorialalice
+# ---
+
+import eelbrain
+import os
+from pathlib import Path
+import copy
+import numpy as np
+from matplotlib import pyplot
+
+# +
+# Data locations
+DATA_ROOT = Path("~").expanduser() / 'Data' / 'Alice'
+PREDICTOR_DIR = DATA_ROOT / 'predictors'
+STIMULI_DIR = DATA_ROOT / 'stimuli'
+TRF_DIR = DATA_ROOT / 'TRFs'
+EPOCH_DIR = DATA_ROOT / 'Epochs'
+
+tstart = -0.1
+tstop = 1
+# -
+
+# Where to save the figure
+DST = DATA_ROOT / 'figures'
+DST.mkdir(exist_ok=True)
+
+# get all subjects
+subjects = [subject for subject in os.listdir(TRF_DIR) if subject.startswith('S') ]
+
+# (1) GET THE ERP RESPONSE TO A WORD ONSET
+cases = []
+for subject in subjects:
+    epochs = eelbrain.load.unpickle(f"{EPOCH_DIR}/{subject}/{subject}_epoched_word.pickle")
```

Comment:
This seems to assume that there are epochs saved here, did you forget to add the script that generates those?

Also, will this work on Windows (with the `/`)? The intended way to use Path objects is like: https://github.com/Eelbrain/Alice/blob/661a010a2ecf56aec48dcb407d07ae1b0df6915a/figures/Auditory-TRFs.py#L60